tests/acceptance/features-test.js

```diff
@@ -13,6 +15,16 @@ module('Acceptance: Features', {
   }
 });
 
+test('features are defined in app config', function() {
+  ok(features.enabled('feature-from-config'), 'feature-from-config is enabled automatically');
```

Comment:
Thanks. I just don't think directly importing a module and calling a method on it is suited to an acceptance-level test.